### PR TITLE
[PHEE-537] helm deploy and helm test in minikube

### PR DESCRIPTION
**Pull request by somanath21**
(https://fynarfin.atlassian.net/browse/PHEE-537)
## Description

* PR title should have jira ticket enclosed in `[]`.<br>
Format: ``` [jira_ticket] description```<br>
ex: [phee-123] PR title.
* Add a link to the Jira ticket.
* Describe the changes made and why they were made.

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [ ] Followed the PR title naming convention mentioned above.

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Created/updated unit or integration tests for verifying the changes made.

- [ ] Added required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,270 +1,181 @@
...
       - run: 
-          name: Ngrok setup
-          command: curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | sudo tee /etc/apt/trusted.gpg.d/ngrok.asc >/dev/null && echo "deb https://ngrok-agent.s3.amazonaws.com buster main" | sudo tee /etc/apt/sources.list.d/ngrok.list && sudo apt update && sudo apt install ngrok
+          name: run minikube 
+          command : |
+            free -m
+            #!/bin/bash
+            #insatll kubectl
+            echo hello 
+            sudo apt-get update
```
Comment (fynmanoj) [inline .circleci/config.yml:19]: please move all the installations to the start of the script. we should plan to create a docker file with all these installations to make the process faster

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,270 +1,181 @@
...
+            cd ..
+            sh orchestration/deployBpmn.sh || echo 'deploy Bpmn done'
+            kubectl get pods -n paymenthub
+            # kubectl describe po -n paymenthub `kubectl get pods -n paymenthub | grep ph-ee-zeebe-ops |cut -d ' ' -f1` || echo ' '
+            # helm list -n paymenthub
+            # sleep 10m
+            # kubectl get -A namespace paymenthub
+            kubectl get pods -n paymenthub
+            # kubectl describe pod ph-ee-elasticsearch-0 -n paymenthub
+            helm test my-ph-ee-g2psandbox -n paymenthub --timeout 5m
```
Comment (fynmanoj) [inline .circleci/config.yml:132]: increase this time out, our tests might take more than 5m

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,270 +1,181 @@
...
+            # until nc -vz ph-ee-zeebe-ops 80; do echo "Waiting for zeebe-ops service"; sleep 2; done;
+            #Deploy BPMN 
+            kubectl port-forward service/ph-ee-zeebe-ops 5000:80 -n paymenthub & #portforward zeebe-ops &'
+            git clone https://github.com/openMF/ph-ee-env-labs.git openMF/ph-ee-env-labs
+            cd openMF/ph-ee-env-labs/orchestration
+            ls
+            sed -i "/HOST=/c\HOST=http://localhost:5000/zeebe/upload" deployBpmn.sh
+            cat deployBpmn.sh
+            cd ..
+            sh orchestration/deployBpmn.sh || echo 'deploy Bpmn done'
```
Comment (fynmanoj) [inline .circleci/config.yml:124]: this is already been added as a job in the chart. do we need to repeat?

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,270 +1,181 @@
...
+            # kubectl create namespace paymenthub
+            # kubectl get -A namespace  
+            helm repo add g2p-sandbox-1-4 https://fynarfin.io/images/ph-ee-g2psandbox-1.4.0/
+            helm install my-ph-ee-g2psandbox g2p-sandbox-1-4/ph-ee-g2psandbox --version 1.4.0 -n paymenthub --timeout 15m
+            # helm repo add g2p-sandbox-1-3-1 https://fynarfin.io/images/ph-ee-g2psandbox-1.3.1/
+            # helm install ph-ee-g2psandbox g2p-sandbox-1-3-1/ph-ee-g2psandbox --version 1.3.1 -n paymenthub 
+
+            git clone -b 7.17 https://github.com/elastic/helm-charts.git elastic/helm-charts
+            # sleep 300
+            cd elastic/helm-charts/elasticsearch/examples/security/
```
Comment (fynmanoj) [inline .circleci/config.yml:96]: these steps are already part of the job, can be removed

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
@somanath21 , Is this PR up to date for merging? , Update to get this merged or close this PR, 
it is been  open for a long time

**Comment by somanath21** [comment]
Ephemeral story is completed

**Pull request closed without merging**